.git-authors

```diff
@@ -7,7 +7,8 @@ pairs:
   km: Klaus Mana
   nw: Nathan Wang; nathan.wang2
   sc: Serena Cao; wcao0630
-  mw: Matt Wills; mattwills8
+  mw: Matt Wills;
+  md: Mark Philip Dawal;
```

Comment:
Add your Gmail address here ~`@gmail`~

Reply:
@bbmoz Mark Philip Dawal used `@gmail.com` so @markphd only need to add the Gmail address.
For @mattwills8 need to remove `;` after `mw: Matt Wills`

Reply:
Ok, should be updated now